### PR TITLE
Re-add lack of plumbing fields

### LINKED ISSUE #1920: Lack of plumbing data fields are missing from the .xls and .csv
The lack of plumbing data fields are missing from the download files. They should be added and they should be added after the column called "Median value ($) of owner-occupied housing units"

### PR DESCRIPTION
**Pull request by mattbowen-usds**
Closes #1920

### CODE REVIEW

**Comment by mattbowen-usds** [comment]
I need to update the pickles it looks like, so switching this to draft

File: data/data-pipeline/data_pipeline/content/config/excel.yml [hunk 1]
```diff
@@ -165,6 +165,12 @@ sheets:
       - score_name: Median value ($) of owner-occupied housing units
         label: Median value ($) of owner-occupied housing units
         format: float
+      - score_name: Share of homes with no kitchen or indoor plumbing (percentile)
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/content/config/excel.yml:168]: This is the opposite order of the other file -- is that intentional?

**Review by lucasmbrown-usds — CHANGES_REQUESTED** [review]

File: data/data-pipeline/data_pipeline/content/config/excel.yml (hunk 1 above)
Comment (mattbowen-usds) [reply data/data-pipeline/data_pipeline/content/config/excel.yml:168]: Good catch! It isn't. I also had to make a bunch of field order changes that are gonna end up causing conflicts, so will fix, but in #1921

**Comment by mattbowen-usds** [comment]
I ended up just moving this into #1921 so that we can see where this column is with all the other correctly moved columns. Closing here and will open a PR for #1921 shortly.

**Pull request closed without merging**